content/02.introduction.md

```diff
@@ -24,69 +24,59 @@ Further, we aim to amplify the true signal out of the vast noise produced by tho
+Individual B-cells have unique membrane-bound antibodies.
+A B-cell is able to bind to an antigen only if its antibody is a match.
+For example, toll-like receptor 3 (TLR3) can detect double-stranded RNA (dsRNA) present during viral replication, allowing it to recognize a viral threat.
+The binding of a pattern recognition receptor (PRR), such as a TLR, to the antigen allows the B-cell to engulf it the virus via endocytocis.
+Once the antigen has been processed and presented on the cell surface, helper T-cells can bind to the MHC-II/antigen complex.
+This second binding step catalyzes the release of cytokines as part of the inflammatory response, as well as the proliferation of active B-cells (plasma cells) capable of producing the antibodies that respond to the antigen.
+Coronaviruses such as SARS-CoV are also able to evade the humoral immune response through other mechanisms, such as inhibiting certain cytokine pathways or down-regulating antigen presentation by the cells [@doi:10.1128/jvi.02501-05].
+Additionally, the cell will produce memory B-cells that can retain information about how to respond to future immune challenges by this pathogen.
+Understanding the humoral response to SARS-CoV-2 is important because this component of the immune response works to reduce the severity of infection in the short term and is also the mechanism through which the body can respond to reexposure.
+
+Research on other coronaviruses provides some evidence of how the innate immune response may proceed in the presence of SARS-CoV-2.
+Antibodies, also known as immunoglobulins, belong to five classes that describe which antigens they bind to.
+Two of the most common immunoglobulin classes are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most common antibodies found in the blood.
+When an antibody binds to an antigen, it forms a complex can then be cleared by the liver or spleen or through the complement cascade.
+Following SARS infection, IgM antibodies were first detected in most patients during the second week following infection and remained detectable until the twelfth week, while IgG antibodies were first detected in the second or third week and remained at close to their peak concentration through the thirteen weeks of the study [@doi:10.1056/NEJM200307313490520].
+A two-year longitudinal study following convalesced SARS patients with a mean age of 29 found that IgG antibodies were detectable in all 56 patients surveyed for at least 16 months, and remained detectable in all but 4 (11.8%) of patients through the full two-year study period [@doi:10.1086/500469].
+Understanding the innate immune response to viral exposure and the mechanisms that allow viruses to evade these mechanisms will be critical for understanding the therapeutic interventions that are being considered for COVID-19.
```

Comment:
what's here is acquired immune response so you can just say "...how the immune response may proceed..." / remove the specification of "innate" (you might consider remove "innate" from the whole subsection here & just find a subheading which says that you're looking at specific immune system features related to SARS-CoV-2)